documentation/high-avail/ha-api-hld.md

```diff
@@ -117,6 +121,11 @@ HA scope is also defined as a SAI object and contains the following SAI attribut
 | SAI_HA_SCOPE_ATTR_FLOW_VERSION | `sai_uint32_t` | The flow version for new flows. |
 | SAI_HA_SCOPE_ATTR_FLOW_RECONCILE_REQUESTED | `bool` | When set to true, flow reconcile will be initiated. |
 | SAI_HA_SCOPE_ATTR_FLOW_RECONCILE_NEEDED | `bool` | (Read-only) If true, flow reconcile is needed. |
+| SAI_HA_SCOPE_ATTR_VIP_V4 | `sai_ip_address_t` | Dedicated IPv4 VIP for DPU HA scope. |
```

Comment:
do you mind to update the description to "IPv4 VIP of the HA scope (Used in DPU-Driven HA mode only)"?

unlike network convergence time, we might want to extend this info to other modes.

Reply:
same as attributes below.

Reply:
done